templates/client/lightblue-client.properties.erb

```diff
@@ -4,7 +4,7 @@ metadataServiceURI=<%= @lbclient_metadata_uri %>
 dataServiceURI=<%= @lbclient_data_uri %>
 useCertAuth=<%= @lbclient_use_cert_auth %>
 <% if(@lbclient_use_cert_auth == true) -%>
-caFilePath=<% if(@lbclient_use_physical_file == true) %>file://<%end%><%= @lbclient_ca_file_path %>
+caFilePath=<% i = @lbclient_ca_certificates.length; @lbclient_ca_certificates.each do |key, hash|; -%><% hash.each do |key,value|; -%><%if (key == 'file') ; -%><% if(@lbclient_use_physical_file == true) ; %>file://<%end;%><%=value%><% end; %><% end; %><% if i > 1 then; -%>,<% end; %><% i = i - 1;-%><% end;%>
```

Comment:
I know having this on one line is hard to read, and I really did try many different ways to make this look cleaner, however all of them resulted in weird spacing and new lines that didn't ultimately work